tests/src/Simple/SimpleTest.targets

```diff
@@ -3,8 +3,8 @@
 
   <PropertyGroup>
     <OutputType Condition="'$(OutputType)' == ''">Exe</OutputType>
-    <OutputPath>$(MSBuildProjectDirectory)\bin\$(Configuration)\</OutputPath>
-    <IntermediateOutputPath>$(MSBuildProjectDirectory)\obj\$(Configuration)\</IntermediateOutputPath>
+    <OutputPath>$(MSBuildProjectDirectory)\bin\$(Configuration)\$(Variant)\</OutputPath>
```

Comment:
It is common to call this `Platform`. Do we have some prior art somewhere where it is called Variant?